SonarQube.Common/SarifUtilities.cs

```diff
@@ -0,0 +1,91 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SarifUtilities.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SonarQube.Common
+{
+    public class SarifUtilities
+    {
+        /// <summary>
+        /// Returns true if the given SARIF came from the VS 2015 RTM Roslyn, which does not provide correct output.
+        /// </summary>
+        public static bool IsSarifFromUnsupportedCompiler(string input)
+        {
```

Comment:
The code is very specific about which tool it is checking for so you could be explicit in the method name e.g. "IsSarifFromVS2015RTM" / "IsSarifFromRoslynV1".

"input" is too generic. "jsonContent" / "json" would be better.